helm/osmcha/values.yaml

```diff
@@ -120,3 +123,10 @@ app:
             }
           }
         }
+
+# OSMCha adiff service
+adiff_service:
+  image:
+    repository: ghcr.io/osmcha/osm-adiff-service
+    tag: db6deae0b0af0aa2e2f00fda2545b4e25c484bc3
+  redis_url: redis://redis-master:6379
```

Comment:
The cron job reuses the same redis db as the main app. Should we introduce some namespace in the keys used by adiff-service to avoid potential conflict? eg: change the key `state` in https://github.com/OSMCha/osm-adiff-service/blob/db6deae0b0af0aa2e2f00fda2545b4e25c484bc3/util/redis-client.js#L9 to `adiff-service:state`

Reply:
I've updated it in the osm-adiff-service code, and updated the image tag here too.